registry/accessing-the-registry.adoc

```diff
@@ -32,13 +32,15 @@ the user must have the `registry-viewer` role. To add this role:
 $ oc policy add-role-to-user registry-viewer <user_name>
 ----
 
-* For writing or pushing images, for example when using the `podman push` command,
-the user must have the `registry-editor` role. To add this role:
+* For writing or pushing images, for example when using the `podman push` command:
+** The user must have the `registry-editor` role. To add this role:
```

Comment:
```suggestion
** The user must have the `registry-editor` role. To add this role, enter:
```

ISG requires language like this when asking the user to enter a command. ("Commands, parameters, and options in instructions," p. 196, latest version)

Reply:
Used 'run the following command' adhering to ISG

Reply:
> ISG requires language like this when asking the user to enter a command. ("Commands, parameters, and options in instructions," p. 196, latest version)

Used 'run the following command' adhering to ISG

Reply:
> ISG requires language like this when asking the user to enter a command. ("Commands, parameters, and options in instructions," p. 196, latest version)

Used 'run the following comman' adhering to ISG.

Reply:
@maxwelldb I have made the suggested changes